CONTRIBUTING.md

```diff
@@ -472,28 +520,32 @@ For information about the syntax, see the [mermaid.js documentation](https://mer
 ```md
 {{< diagram >}}
 flowchart TB
-  This --> That
-  That --> There
+This --> That
+That --> There
 {{< /diagram >}}
```

Comment:
The linter is removing spaces inside of shortcodes that need to be there.

Reply:
> The linter is removing spaces inside of shortcodes that need to be there.

I can't tell that indentation matters here.  E.g. in content/telegraf/v1.17/concepts/aggregator_processor_plugins.md:13 and content/enterprise_influxdb/v1.8/concepts/clustering.md:20, removing the indent generates the same HTML. That said, I like the indentation readability.

I've done some research, but haven't found a way to globally ignore handlebars-style sections in `.md` files. You can add a prettier-ignore statement individually, but that's no fun.